sphinx_doc/setup_guide.rst

```diff
@@ -1,328 +1,246 @@
+   # Set to start at boot time: 
+   sudo chkconfig memcached on 
 
 Database
 --------
 
-Using PostgreSQL as Database Backend:: 
+Install PostgreSQL::
 
-    # install PostgresSQL
+    # Add line to yum repository: 
+    echo 'exclude=postgresql*' | sudo tee -a /etc/yum.repos.d/CentOS-Base.repo
 
-    postgres=# create user django; 
-    postgres=# create database django;
-    postgres=# grant all on database django to django;
-    # connect to database django
-    postgres=# \c django 
-    # create extension hstore
-    postgres=# create extension hstore;
-
-    # config in pg_hba.conf
-    pip install psycopg2==2.6
-
-* Install PostgreSQL ( `postgresql reference`_ )
-
-  .. _postgresql reference: https://www.digitalocean.com/community/tutorials/how-to-install-and-use-postgresql-on-centos-7
-* Create the user and database for this application
-* Allowing connection from the database user by setting your ``/var/lib/pgsql/9.x/data/pg_hba.conf``
-* Install pip package psycopg2. Path of pg_config binary may need to be exported. 
+    # Install the PostgreSQL Global Development Group (PGDG) RPM file:
+    sudo yum -y localinstall http://yum.postgresql.org/9.5/redhat/rhel-6-x86_64/pgdg-centos95-9.5-2.noarch.rpm
+    
+    # Install PostgreSQL 9.5:
+    sudo yum -y install postgresql95-server postgresql95-contrib postgresql95-devel
+    
+    # Initialize (uses default data directory: /var/lib/pgsql):
+    sudo service postgresql-9.5 initdb   
+    
+    # Startup at boot:
+    sudo chkconfig postgresql-9.5 on
+    
+    # Control:
+    # sudo service postgresql-9.5 <command>
+    # 
+    # where <command> can be:
+    #  
+    #     start   : start the database.
+    #     stop    : stop the database.
+    #     restart : stop/start the database; used to read changes to core configuration files.
+    #     reload  : reload pg_hba.conf file while keeping database running. 
+    
+    # Start:
+    sudo service postgresql-9.5 start
 
-::
+    #
+    #  (To remove everything: sudo yum erase postgresql95*)
+    #
+    
+    # Create django database and user:
+    sudo su - postgres
+    psql
+    
+    # At the prompt 'postgres=#' enter:
+    create database django;
+    create user django;
+    grant all on database django to django;
+    
+    # Connect to django database:
+    \c django
+    
+    # Create extension hstore:
+    create extension hstore;
 
+    # Config in pg_hba.conf:
+    cd <virt-env> 
+    export PATH=/usr/pgsql-9.5/bin:$PATH
     pip install psycopg2==2.6
-    export PATH=/usr/pgsql-9.x/bin:"$PATH"
 
-Migration db schema to PostgreSQL 
+ 
+Migrate Schema to to PostgreSQL
+------------------------------- 
 
-::
+Run migrate::
 
+    cd <virt-env>
```

Comment:
Basic question, is this essentially the same as running 
source <virt-env>/py-2.7/bin/activate
then running manage.py migrate?